prime-dotnet-webapi/ViewModels/Logging/FrontEndLogViewModel.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using Prime.Models;
+
+namespace Prime.ViewModels
+{
+    public class FrontEndLogViewModel
+    {
+        public string Msg { get; set; }
+        public string Data { get; set; }
```

Comment:
Seems a bit simpler to have the log type in the view model rather than as a route parameter.

Reply:
The reason i chose this path is it reduced the work and code needed in the front end. Since i'm just using the default object generated in our logging service and sending that up as a sudo view model. This also reflects the naming Msg/Data.